package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "electron-spellchecker",
+  "name": "@kwonoj/electron-spellchecker",
```

Comment:
Instead of publishing to npm you could change the dependency declaration in `package.json` to use your fork